src/components/DRB/DRBCard.tsx

```diff
@@ -22,18 +23,20 @@ const DRBCard = ({ drbResult }: DRBCardProps) => {
 
   return (
     <Card backgroundColor="ui.bg.default" p="xs">
-      <CardContent>
+      <CardHeading level="h3" size="heading6" mb="0">
         <DSLink
           href={drbResult.url}
           target="_blank"
           isUnderlined={false}
           fontSize="desktop.body.body2"
           display="inline-block"
           mb="s"
+          lang={drbResult.language}
```

Comment:
I think this might be worth adding when the language is something besides English.

Reply:
Good call! Added a check to only render the language when it's not english.